<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibrarySettings</class>
 <widget class="QWidget" name="LibrarySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The library settings need a restart before they show any effect.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="documentLibraryTab">
      <attribute name="title">
       <string>Documents</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Folder Whitelist</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>List of folders that should be used for the system library document list. If the list is empty all Nepomuk known documents will be listed.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lwSystemWhiteList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbAddSystemFolder">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbRemoveSystemFolder">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Nepomuk Documents Types</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="cbShowDocumentsType">
            <property name="toolTip">
             <string>These are all files that might make a document including sourcecode etc</string>
            </property>
            <property name="text">
             <string>Documents</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowPaginatedTextDocument">
            <property name="toolTip">
             <string>These are rich text documents like pdf, odt, doc etc</string>
            </property>
            <property name="text">
             <string>PaginatedTextDocument</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowSpreadSheetsType">
            <property name="toolTip">
             <string>These are Spreadsheets like xls,ods</string>
            </property>
            <property name="text">
             <string>Spreadsheets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowMindMaps">
            <property name="toolTip">
             <string>These are MindMap files</string>
            </property>
            <property name="text">
             <string>Mind Maps</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowPlainTextDocumentsType">
            <property name="toolTip">
             <string>These are all kind of text documents like .txt</string>
            </property>
            <property name="text">
             <string>PlainTextDocuments</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>210</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="cbShowOnlyFileNotes">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>List all Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>List Document Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_5">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>List Project Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>257</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
